.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
 install:
 -  npm install
 -  npm install -g grunt-cli
--  pip install -r requirements.txt
+-  pip install -r dev_requirements.txt
```

Comment:
The way django-compressor is currently implemented, we need it for production as well, so I would rather keep both requirements.txt and dev-requirements.txt side by side.